gems/slosilo/CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# v3.0.1
```

Comment:
Mind adding a note to the top of this file to direct users to add Slosilo changes to the main `Changelog.md`?

Example from Conjur-Rack: https://github.com/cyberark/conjur/blob/master/gems/conjur-rack/CHANGELOG.md

Reply:
Done